LocalFolder/explainer.md

```diff
@@ -0,0 +1,159 @@
+# ApplicationData.LocalFolder Access From Microsoft Store PWA 
+
+Author: [Lu Huang](https://github.com/luhuangMSFT)
+
+## Status of this Document
+
+This document is a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+
+* This document status: **Active**
+* Expected venue: [Microsoft Edge Explainers](https://github.com/MicrosoftEdge/MSEdgeExplainers)
+* **Current version: this document**
+
+## Introduction
+[UWP apps](https://learn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide) often make use of the [`ApplicationData.LocalFolder`](https://learn.microsoft.com/en-us/uwp/api/windows.storage.applicationdata.localfolder) API for access to local storage. This API manages storage in a system file directory unique to the [Package Family Name (PFN)](https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/package-identity-overview#package-family-name) of the installed app package. A UWP app distributed by the [Microsoft Store](https://apps.microsoft.com/home) may be replaced with a PWA app under the same PFN. When this happens, existing files in the `LocalFolder` directory become inaccessible to the PWA. 
```

Comment:
> existing files in the `LocalFolder` directory become inaccessible to the PWA

Why? What's different that affects this?

Reply:
"Sandboxing" of PWA vs Native App environment does not allow the PWA to access the hard drive location of he native app it is replacing. That is the way it exists today *without* this feature. For data migration purposes, this feature would allow the PWA to access this location for the purposes of data migration.

Reply:
Edited to explain that this is because WinRT APIs are not exposed to the Web once a UWP app gets replaced with a PWA app package.